src/pear/config.vsh.yaml

```diff
@@ -0,0 +1,145 @@
+functionality:
+  name: pear
+  description: "PEAR is an ultrafast, memory-efficient and highly accurate pair-end read merger"
+  info:
+    keywords: [ "pair-end", "read", "merge" ]
+    homepage: https://cme.h-its.org/exelixis/web/software/pear
+    documentatoin: https://cme.h-its.org/exelixis/web/software/pear/doc.html
+    reference: doi:10.1093/bioinformatics/btt593
+    licence: "['Creative Commons Attribution-NonCommercial-ShareAlike 3.0 Unported']"
```

Comment:
This is a list in a string. either write `license: "Creative Commons ..."` or `license: ["Creative Commons ..."]`.

Also, I plan on adding a validator on the license, so using a standardized formatting of the licenses probably makes sense. E.g. https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository -- so this would become `license: CC-BY-NC-SA-3.0`.

Reply:
You are correct, Something went wrong with saving my changes and forgot to redo this one :).